notebooks/logging-examples/logging-feature-plots.ipynb

```diff
@@ -0,0 +1,466 @@
+   "source": [
+    "`Rubicon_ml` makes it easy to log plots with features and artifacts. In this example we'll walk through creating a feature dependency plot using the `shap` package and saving it to an artifact."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e6c4ba0",
+   "metadata": {},
+   "source": [
+    "## Set up\n",
+    "First lets create a Rubicon `project` and create a pipeline with `rubicon_ml.sklearn.pipeline`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "932d5223",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "RubiconPipeline(project=<rubicon_ml.client.project.Project object at 0x18500f670>,\n",
+       "                steps=[('gradientboostingregressor',\n",
+       "                        GradientBoostingRegressor(random_state=1))])"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import shap\n",
+    "import sklearn\n",
+    "from sklearn.datasets import load_wine\n",
+    "from rubicon_ml import Rubicon\n",
+    "\n",
+    "from sklearn.preprocessing import StandardScaler\n",
+    "from sklearn.ensemble import GradientBoostingRegressor\n",
+    "from rubicon_ml.sklearn import make_pipeline\n",
+    "\n",
+    "rubicon = Rubicon(persistence=\"memory\")\n",
+    "project = rubicon.get_or_create_project(\"Logging Feature Plots\")\n",
+    "\n",
+    "X, y = load_wine(return_X_y=True)\n",
+    "\n",
+    "reg =  GradientBoostingRegressor(random_state=1)\n",
+    "pipeline=make_pipeline(project, reg)\n",
+    "\n",
+    "pipeline.fit(X, y)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef941fa1",
+   "metadata": {},
+   "source": [
+    "## Generating Data\n",
+    "After fitting the pipeline, using `shap.Explainer` we can generate `shap` values to later plot. For more information on generating `shap` values with `shap.explainer`, check the documentation [here](https://shap.readthedocs.io/en/latest/generated/shap.Explainer.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b158cb8c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "explainer = shap.Explainer(pipeline[0])\n",
+    "shap_values = explainer.shap_values(X)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "596f75d3",
+   "metadata": {},
+   "source": [
+    "## Plotting\n",
+    "The generated `shap_values` from the above cell can be passed to a `shap.depence_plot` to generate a dependence plot. `pl.gcf()` allows the plot generated by `shap` to be saved to a variable. Using the `matplotlib` and  `io` libraries, shap plots can be saved to a `byte` representation. Here, a feature and its plot are both logged with`rubicon_ml.Features` and `rubuicon_ml.Artifact` respectively. These `features` and `artifacts` are logged to the same `rubicon_ml.Experiment` which are being kept track of with an experiment tag.  "
```

Comment:
"which are being kept track of with an experiment tag" should be removed from the end of that last sentence